project/app/src/main/java/com/fillr/example/integration/activity/ExampleWebViewHeadlessActivity.java

```diff
@@ -134,16 +134,13 @@ private void setupFillr(WebView webView) {
 
         //Step 3. Set the FillMode. We support two different modes headless and headful.
         fillr.setFillMode(Fillr.FillMode.HEADLESS);
+
         //Step 4. Set the data provider. This is called when,
         //  - The form in question mutates
         //  - When the page finishes loading.
         fillr.profileDataListener(profileDataListener);
 
-        //Step 5 - Setup Affiliate Url Redirection (optional feature)
-        config.setAffiliateProvider(FillrConfig.AFFILIATE_PROVIDER_VIGLINK, "Affiliate API Key Provided by Fillr");

```

Comment:
Rather than being removed entirely it's still necessary to call `config.setAffiliateEnabled(true);` here.  Otherwise the SDK won't request affiliate links, even if the dev-key has access to them.

Reply:
updated!